src/BehaviorsSDKNative/Microsoft.Xaml.Interactions.Design/MetadataTableProvider.cs

```diff
@@ -1,20 +1,28 @@
 ﻿// Copyright (c) Microsoft. All rights reserved. 
 // Licensed under the MIT license. See LICENSE file in the project root for full license information. 
 
-using System;
 using System.ComponentModel;
+
+#if SurfaceIsolation
+using Microsoft.VisualStudio.DesignTools.Extensibility.Metadata;
+using Microsoft.VisualStudio.DesignTools.Extensibility.PropertyEditing;
+using Editors = Microsoft.VisualStudio.DesignTools.Extensibility.PropertyEditing.Editors;
+using Microsoft.Xaml.Interactions.Design.Properties;
+#else
+using System;
 using Microsoft.Windows.Design.Metadata;
 using Microsoft.Windows.Design.PropertyEditing;
-using Microsoft.Windows.Design.PropertyEditing.Editors;
-using Microsoft.Xaml.Interactions.Core;
+using Editors = Microsoft.Windows.Design.PropertyEditing.Editors;
 using Microsoft.Xaml.Interactions.Design.Properties;
-using Microsoft.Xaml.Interactions.Media;
+#endif
+
+[assembly: ProvideMetadata(typeof(Microsoft.Xaml.Interactivity.Design.MetadataTableProvider))]
 
```

Comment:
Aren't these different for SI vs DI? They should probably go each into the respective #if #else

Reply:
I kept the same namespace for the DesignTools project to reduce the differences between the projects. I believe the benefit of using the same namespace outweighs that of observing the convention. 
The ProvideMetadata attribute refers to the class defined in that file.

Reply:
How about we drop .Design altogether then? That would minimize confusion.

---
In reply to: [382725850](https://github.com/microsoft/XamlBehaviors/pull/172#discussion_r382725850) [](ancestors = 382725850)

Reply:
While it’s a good suggestion, I believe we should leave it as is. My reasoning is that having the designer specific code in a sub-namespace may prove to be useful down the track. I’ve added a brief explanatory comment above the namespace in all of the MetadataTableProvider.cs files.